### PR TITLE
Fixing links broken with change from com.appboy to com.braze

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

### CODE REVIEW

**Comment by bre-fitzgerald** [comment]
Hey @davidbielik, it was brought to our attention yesterday that some of the kdoc links in the Android docs are broken after the switch from `com.appboy` to `com.braze`. I've gone through and updated any broken links to where I think they're supposed to lead now, but would you or someone on your team be able to review this PR and confirm? Thanks! 

cc: @KellieHawks

**Comment by davidbielik** [comment]
> Hey @davidbielik, it was brought to our attention yesterday that some of the kdoc links in the Android docs are broken after the switch from `com.appboy` to `com.braze`. I've gone through and updated any broken links to where I think they're supposed to lead now, but would you or someone on your team be able to review this PR and confirm? Thanks!
> 
> cc: @KellieHawks

thanks! @radixdev? 👀

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/network_offline_mode_integration.md [hunk 1]
```diff
@@ -34,4 +34,4 @@ Braze.setOutboundNetworkRequestsOffline(true)
 {% endtab %}
 {% endtabs %}
 
-[1]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy/-appboy/set-outbound-network-requests-offline.html
+[1]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/Appboy.html#setOutboundNetworkRequestsOffline-boolean-
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/network_offline_mode_integration.md:37]: "javadocs" should never be in the docs anymore. It should always be "kdoc" in the link.

https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze/-braze/-companion/outbound-network-requests-offline.html?query=var%20outboundNetworkRequestsOffline:%20Boolean

If you see "javadocs" somewhere, @ someone on the team (or me) to update them.

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization/default_feed.md [hunk 2]
```diff
@@ -207,10 +207,10 @@ To fully customize the network error behavior, you can extend the [`ContentCards
 
 [49]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.contentcards/-content-cards-fragment/index.html
 [52]: https://github.com/Appboy/appboy-android-sdk/blob/2e386dfa59a87bfc24ef7cb6ff5adf6b16f44d24/android-sdk-ui/src/main/res/values/styles.xml#L522-L530
-[53]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.contentcards/-appboy-empty-content-cards-adapter/index.html
-[54]: https://github.com/Appboy/appboy-android-sdk/blob/master/android-sdk-ui/src/main/java/com/braze/ui/contentcards/ContentCardsFragment.java
-[50]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.contentcards/-appboy-empty-content-cards-adapter/index.html
-[36]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.models.cards/-card/get-extras.html
+[53]: https://github.com/Appboy/appboy-android-sdk/blob/master/android-sdk-ui/src/main/java/com/braze/ui/contentcards/adapters/ContentCardAdapter.kt
+[54]: https://github.com/Appboy/appboy-android-sdk/blob/master/android-sdk-ui/src/main/java/com/appboy/ui/AppboyContentCardsFragment.java
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/customization/default_feed.md:211]:
```suggestion
[54]: https://github.com/Appboy/appboy-android-sdk/blob/master/android-sdk-ui/src/main/java/com/braze/ui/contentcards/ContentCardsFragment.kt
```

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/integration.md [hunk 3]
```diff
@@ -197,7 +197,7 @@ If a card is already marked as dismissed, it cannot be marked as dismissed again
 [30]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.models.cards/-banner-image-card/index.html
 [31]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.models.cards/-captioned-image-card/index.html
 [32]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.models.cards/-text-announcement-card/index.html
-[41]: https://github.com/Appboy/android-sdk/blob/9a091979b4cbaff7f935c2cae03043a944c3ab53/android-sdk-base/src/main/java/com/appboy/models/cards/ShortNewsCard.java
+[41]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.models.cards/-short-news-card/index.html
 [2]: http://developer.android.com/guide/components/fragments.html
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/integration.md:201]:
```suggestion
[2]: https://developer.android.com/guide/components/fragments.html
```

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/fireos/integration.md [hunk 4]
```diff
@@ -117,10 +117,10 @@ Braze does not recommend using manual registration, but if you need to handle AD
 <!-- This will disable automatic registration for ADM via the Braze SDK-->
 <bool name="com_braze_push_adm_messaging_registration_enabled">false</bool>
 ```
-Next, use [registerPushToken][37] to pass your user's ADM `registration_id` to Braze:
+Next, use [`Braze.setRegisteredPushToken()`](https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze/-braze/registered-push-token.html) to pass your user's ADM `registration_id` to Braze:
 
 ```java
-Braze.getInstance(context).registerPushToken(registration_id);
+Braze.getInstance(context).Braze.setRegisteredPushToken(registration_id);
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/fireos/integration.md:123]:
```suggestion
Braze.getInstance(context).setRegisteredPushToken(registration_id);
```

Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/push_notifications/fireos/integration.md:123]: Please add a `kotlin` equivalent for this line.

The code is:

```kotlin
Braze.getInstance(context).registeredPushToken = registration_id
```

**Review by radixdev — DISMISSED** [review]

**Review by lydia-xie — APPROVED** [review]
lgtm!

**Pull request merged**